src/main/java/tech/autodirect/api/database/TableUsers.java

```diff
@@ -43,22 +45,25 @@ public void addUser(
         int creditScore,
         double downPayment,
         double budgetMonthly
-    ) throws SQLException {
-        PreparedStatement stmt = this.dbConn.prepareStatement(
-            "INSERT INTO " + this.schemaName + "." + this.tableName + " VALUES (?, ?, ?, ?, ?);"
-        );
-        stmt.setString(1, userId);
-        stmt.setInt(2, creditScore);
-        stmt.setBigDecimal(3, BigDecimal.valueOf(downPayment));
-        stmt.setBigDecimal(4, BigDecimal.valueOf(budgetMonthly));
-        stmt.setString(5, TableOffersInterface.createTableName(userId));
-
-        // execute and close the above SQL statement
-        stmt.executeUpdate();
-        stmt.close();
-
-        // TODO: What happens if user already exists?
-        // TODO: What happens if there is no offers table for this user yet (which is usually the case)?
+    ) throws SQLException, ClassNotFoundException {
+        if (!userExists(userId)) {
```

Comment:
Why are we still doing this?
It should throw an error

Reply:
Let's explicitly do the check in SvcUserLogin, and create a table there explicitly.